services/notification/notifiers/mixins/message/helpers.py

```diff
@@ -116,6 +116,53 @@ def make_metrics(before, after, relative, show_complexity, yaml):
     return "".join(("|", coverage, complexity, icon))
 
 
+def make_patch_only_metrics(before, after, relative, show_complexity, yaml):
+    if after is None:
+        # e.g. missing flags
+        coverage = " `?` |"
+
+    elif after is False:
+        # e.g. file deleted
+        coverage = " |"
+
+    else:
+        layout = " `{relative}` |"
+        coverage = layout.format(
+            relative=format_number_to_str(
+                yaml, relative.coverage if relative else 0, style="{0}%", if_null="\xF8"
+            ),
+        )
+    return "".join(("|", coverage))
+
+
+def get_metrics_method(hide_project_coverage):
```

Comment:
[very nit] I think that technically both `make_patch_only_metrics` and `make_metrics` are functions, not methods